torch/autograd/profiler.py

```diff
@@ -208,29 +237,40 @@ def export_chrome_trace(self, path):
             f.truncate()
             f.write("]")
 
-    def key_averages(self, group_by_input_shapes=False):
+    def key_averages(self, group_by_input_shapes=False, group_by_stack_n=0):
         """Averages all function events over their keys.
 
-        @param group_by_input_shapes The key would become
-        (event name, input dimensions) rather than just event name.
-        This is useful to see which dimensionality contributes to the runtime
-        the most and may help with dimension specific optimizations or
-        choosing best candidates for quantization (aka fitting a roof line)
+        Arguments:
+            group_by_input_shapes: The key would become
+            (event name, input dimensions) rather than just event name.
+            This is useful to see which dimensionality contributes to the runtime
```

Comment:
nit: which input shapes. I know it's not this PR, but still, dimensionality is the wrong work here.